react/ShippingOption.tsx

```diff
@@ -14,61 +17,95 @@ interface EstimateDeliveryOption {
 interface Props {
   onSelectDeliveryOption?: () => void
   onDeselectDeliveryOption?: () => void
-  deliveryOption: DeliveryOption
+  shippingOption: any
```

Comment:
I don't know how I could typefy this. I have tried both:

```
shippingOption: DeliveryOption | PickupOption

shippingOption: DeliveryOption & PickupOption
```

But I couldn't solve the errors that these options caused.

Reply:
🤔 which errors appeared?